webapi/src/main/scala/org/knora/webapi/slice/admin/domain/model/Permission.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright © 2021 - 2024 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.slice.admin.domain.model
+
+sealed trait Permission {
+  def token: String
```

Comment:
**comment (non-blocking):** I find the term `token` somewhat confusing because it makes me think of JWTs. But I can't think of anything better for now. Maybe in the future we can try to think of something more descriptive.

Reply:
I am happy to hear any suggestions.

The `token` contains now the value which is actually written as a token in the `hasPermission` property:

For example the full name for all administrative Permissions like `ProjectAdminGroupAllPermission` or the  abbreviation for the object access permissions like `CR`. Other property names I have considered were: `abbreviation`, `code`, `symbol`, `name`, wdyt?

Reply:
I spontaneously like `code` best. But at the end of the day, I'm ok with either of those as well as with `token`